phpBB/includes/ucp/ucp_notifications.php

```diff
@@ -70,21 +71,25 @@ public function main($id, $mode)
 								* @var	array	type_data		The notification type data
 								* @var	array	method_data		The notification method data
 								* @var	bool	is_set_notify	The notification is set or not
+								* @var	bool	is_available	The notification is available or not
+								* @var	array	subscriptions	The subscriptions data
 								*
 								* @since 3.2.10-RC1
 								*/
```

Comment:
Needs `@changed` info

Reply:
@marc1706 ,
You mean `@changed 3.2.10-RC1 Added is_available and subscriptions`
i did not add this coz the event was created in same RC version.

I you still want me to add it , let me know.

Best regards 👍

Reply:
Also should I also add `@since 3.3.1-RC1` .

Best regards 👍

Reply:
Ah sorry, I missed that. In that case, yes, please instead add a `@since 3.3.1-RC1` as well.

Reply:
Thanks 😄 

I have done the required changes.

Best regards 👍